.vscode/launch.json

```diff
@@ -25,6 +25,24 @@
       "autoAttachChildProcesses": true,
       "sourceMaps": true,
       "preLaunchTask": "npm: build-vscode"
+    },
+    {
+      "type": "extensionHost",
+      "request": "launch",
+      "name": "Debug VSCode Tests",
+      "runtimeExecutable": "${execPath}",
+      "args": [
+        "--extensionDevelopmentPath=${workspaceRoot}/packages/vscode-extension",
+        "--extensionTestsPath=${workspaceRoot}/packages/vscode-extension/dist/e2e_tests/testRunnerDebug"
+      ],
+      "outFiles": [
+        "${workspaceRoot}/packages/vscode-extension/**/*.js",
+        "${workspaceRoot}/packages/language-support/**/*.js",
+        "${workspaceRoot}/packages/schema-poller/**/*.js"
```

Comment:
Add `"${workspaceRoot}/packages/language-server/**/*.js"` here so we can debug the events in the language server as well

Reply:
done :)